### PR TITLE
RESTWS-776: Enhance fulfiller details sub resource to allow updating accession number of an order

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetails2_3.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_3;
+
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2.FulfillerDetails2_2;
+
+public class FulfillerDetails2_3 extends FulfillerDetails2_2 {
```
Comment (mseaton) [inline omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetails2_3.java:5]: Javadoc?

File: omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.api.PageableResult;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingSubResource;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2.FulfillerDetails2_2;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2.OrderResource2_2;
+
+@SubResource(parent = OrderResource2_2.class, path = "fulfillerdetails", supportedClass = FulfillerDetails2_2.class, supportedOpenmrsVersions = {
```
Comment (mseaton) [inline omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3.java:20]: Shouldn't the "supportedClass" be FullfillerDetails2_3.class?

File: omod-2.3/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3Test.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_3;
+
+import org.openmrs.Order;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+import org.openmrs.module.webservices.rest.web.v1_0.RestTestConstants2_3;
+
```
Comment (mseaton) [inline omod-2.3/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3Test.java:16]: Javadoc?

File: omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+import org.openmrs.module.webservices.rest.web.annotation.SubResource;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.api.PageableResult;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingSubResource;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2.FulfillerDetails2_2;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2.OrderResource2_2;
+
```
Comment (mseaton) [inline omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3.java:19]: Javadoc?

File: omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3.java (hunk 2 above)
Comment (mogoodrich) [reply omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3.java]: Good catch, thanks!

File: omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetails2_3.java (hunk 1 above)
Comment (mogoodrich) [reply omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetails2_3.java:8]: Added

File: omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3.java (hunk 4 above)
Comment (mogoodrich) [reply omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_3/FulfillerDetailsResource2_3.java:19]: Added!

**Pull request merged**